RZTransitions-Demo/RZTransitions-Demo/ViewControllers/RZSimpleViewController.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  RZSimpleViewController.swift
+//  RZTransitions-Demo
+//
+//  Created by Eric Slosser on 11/13/15.
+//  Copyright 2015 Raizlabs and other contributors
+//  http://raizlabs.com/
+//
+//  Permission is hereby granted, free of charge, to any person obtaining
+//  a copy of this software and associated documentation files (the
+//  "Software"), to deal in the Software without restriction, including
+//  without limitation the rights to use, copy, modify, merge, publish,
+//  distribute, sublicense, and/or sell copies of the Software, and to
+//  permit persons to whom the Software is furnished to do so, subject to
+//  the following conditions:
+//
+//  The above copyright notice and this permission notice shall be
+//  included in all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+//  EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+//  MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+//  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+//  LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+//  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+//  WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import UIKit
+
+@objc(RZSimpleViewController)
+class RZSimpleViewController: UIViewController, RZTransitionInteractionControllerDelegate
+{
+    @IBOutlet weak var popButton: UIButton?
+    @IBOutlet weak var pushButton: UIButton?
+    @IBOutlet weak var modalButton: UIButton?
+    @IBOutlet weak var collectionViewButton: UIButton?
+
+    var pushPopInteractionController: RZTransitionInteractionController?
+    var presentInteractionController: RZTransitionInteractionController?
+
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: NSBundle?) {
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    @IBAction func pushNewViewController(_: AnyObject)
+    {
```

Comment:
Picky, and though in direct violation of the wisdom of the prophets Kernighan and Ritchie, opening `{`'s should be on the same line as the declaration.